### PR TITLE
{cae}[dummy/dummy] LS-PrePost v4.6

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.6-centos6-03Apr2019.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+easyblock = "Tarball"
+
+name = 'LS-PrePost'
+version = '4.6'
+versionsuffix = '-centos6-03Apr2019'
```
Comment (smoors) [inline easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.6-centos6-03Apr2019.eb]: the date is not related to the centos version but to the the release date? then maybe replace this with:
```
release_date = '03Apr2019'
versionsuffix = '-centos6'
```
and also update the easyconfig name.

File: easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.6-centos6-03Apr2019.eb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+homepage = 'http://lstc.com/products/ls-prepost'
+description = """LS-PrePost is an advanced pre and post-processor that is delivered free with LS-DYNA."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://ftp.lstc.com/anonymous/outgoing/lsprepost/%(version)s/linux64/']
+sources = ['lsprepost-%%(version)s_mesa%s.tgz' % versionsuffix]
```
Comment (smoors) [inline easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.6-centos6-03Apr2019.eb]: in accordance with above, maybe replace with:
```
sources = ['lsprepost-%%(version)s_mesa%s-%s.tgz' % (versionsuffix, release_date)]
```

File: easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.6-centos6-03Apr2019.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.6-centos6-03Apr2019.eb]: Thanks, I'm not sure why I thought it was some build version suffix. It being a release date seems much more reasonable. I have done as you suggested.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic413 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/9b0b566745e48a6881693d99696a6ae3 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @Micket!

**Pull request merged**